### PR TITLE
Add user.set_password action + tests.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: openslides_backend/action/user/set_password.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        set hashed password and set default password if set_as_default is True.
+        """
+        password = instance.pop("password")
+        set_as_default = False
+        if "set_as_default" in instance:
+            set_as_default = instance.pop("set_as_default")
+        hashed_password = self.auth.hash(password)
+        instance["password"] = hashed_password
+        if set_as_default:
+            instance["default_password"] = hashed_password
```
Comment (jsangmeister) [inline openslides_backend/action/user/set_password.py:32]: `default_password` must contain the password in clear text

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/user/set_password.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/user/set_password.py]: Done.

**Review by jsangmeister — APPROVED** [review]
Please squash and then it can be merged

**Comment by jsangmeister** [comment]
Done.

**Pull request merged**